app/src/main/java/lt/vilnius/tvarkau/dagger/module/NewApi.kt

```diff
@@ -0,0 +1,12 @@
+package lt.vilnius.tvarkau.dagger.module
+
+/**
+ * @author Martynas Jurkus
+ */
+
+import javax.inject.Qualifier
+import kotlin.annotation.AnnotationRetention.RUNTIME
+
+@Qualifier
+@Retention(RUNTIME)
+annotation class NewApi
```

Comment:
It's purpose is to distinct several non-unique objects for injection.

For example in this project Dagger providers 2 Retrofit instances, 2 OkHttpClient instances. 
One from LegacyApiModule, another from ApiModule.

We could create similar annotation like `OldApi`, but it's redundant in my opinion.